_posts/en/newsletters/2023-03-22-newsletter.md

```diff
@@ -0,0 +1,119 @@
+---
+title: 'Bitcoin Optech Newsletter #243'
+permalink: /en/newsletters/2023/03/22/
+name: 2023-03-22-newsletter
+slug: 2023-03-22-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+*No significant news this week was found on the Bitcoin-Dev or
+Lightning-Dev mailing lists.*
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Xapo Bank supports Lightning:**
+  Xapo Bank [announced][xapo lightning blog] its customers can now send outgoing
+  Lightning payments from the Xapo Bank mobile apps, using underlying infrastructure from Lightspark.
+
+- **TypeScript library for miniscript descriptors released:**
+  The TypeScript-based [Bitcoin Descriptors Library][github descriptors library]
+  has support for [PSBTs][topic psbt], [descriptors][topic descriptors],
+  [miniscript][topic miniscript], and signing support including some hardware
+  signing devices.
+
+- **Breez Lightning SDK announced:**
+  In a recent [blog post][breez blog], Breez announced the open source [Breez
+  SDK][github breez sdk] for mobile developers who want to integrate Bitcoin and
+  Lightning payments. The SDK includes support for [Greenlight][blockstream
+  greenlight], Lightning Service Provider (LSP) features, and other services.
+
+- **PSBT-based exchange OpenOrdex launches:**
+  The [open source][github openordex] exchange software allows sellers to create
+  an order book of Ordinal satoshis using [PSBTs][topic psbt] and buyers to sign
+  and broadcast to complete the trade.
+
+- **BTCPay Server coinjoin plugin released:**
+  The Wasabi Wallet [announcement][wasabi blog] notes that any BTCPay Server
+  merchant can activate the optional plugin which supports the
+  [WabiSabi][news194 wabisabi] protocol for [coinjoins][topic coinjoin].
```

Comment:
Changing this internal link to newsletter 101, which I think has a better description (and no distraction about payjoins).

Reply:
ACK on this and the other client/services section updates.